src/test/regress/input/multi_alter_table_statements.source

```diff
@@ -162,7 +162,54 @@ FROM
 ORDER BY attnum;
 \c - - - :master_port
 
+-- check we don't intercept DDL commands if propagation is turned off
+SET citus.enable_ddl_propagation to false;
+-- table rename statement can be performed now
+ALTER TABLE lineitem_alter RENAME TO lineitem_renamed;
+-- verify rename is performed
+SELECT relname FROM pg_class WHERE relname = 'lineitem_alter' or relname = 'lineitem_renamed';
+-- revert it to original name
+ALTER TABLE lineitem_renamed RENAME TO lineitem_alter;
+
+-- this column is added to master table and not workers
+ALTER TABLE lineitem_alter ADD COLUMN not_added_column int;
+-- verify newly added column is not present in a worker shard
+\c - - - :worker_1_port
+SELECT not_added_column FROM lineitem_alter_103000 LIMIT 0;
+\c - - - :master_port
+
+-- ddl propagation flag is reset to default, disable it again
+SHOW citus.enable_ddl_propagation;
```

Comment:
that is to verify it is a session variable. It got reset when you open a new session.